CHANGELOG.md

```diff
@@ -20,7 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 -
--
+- Add parameter `save_ts` to fitting pipelines ([#73](https://github.com/etna-team/etna/pull/73))
```

Comment:
May be "Add parameter `save_ts` to pipeline's method `fit`" or sth like this -- now sounds strange